bindings/go/http/batch.go

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
nit: file name app.go?

Reply:
I'm fine with app.go.  Is app.go super standard for Golang?

Reply:
We have app.go in all of our building block examples.